definition-src/Game/LambdaHack/Definition/Defs.hs

```diff
@@ -1,9 +1,9 @@
 {-# LANGUAGE DeriveGeneric #-}
 -- | Basic types for content definitions.
 module Game.LambdaHack.Definition.Defs
-  ( GroupName, displayGroupName
+  ( GroupName(..), displayGroupName
   , ContentId, contentIdIndex
-  , ContentSymbol, displayContentSymbol
+  , ContentSymbol , displayContentSymbol, toContentSymbol
```

Comment:
Once you remve the uses of `toContentSymbol` from the code, please remove the export of `toContentSymbol` here, too.